app/controllers/api/LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2012 Brian Ploetz (bploetz@gmail.com)
```

Comment:
But this is not part of the versionist distribution? If you just use the Gem, there is no reason to supply a license file here. Dedicated license files are only needed if you copy source from third parties, for example if `app/controllers/api/openid_connect/authorizations_controller.rb` was copy&pasted from some application.

I don't think that's the case here.